sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/immunizations/CreateNewImmunizationSteps.java

```diff
@@ -0,0 +1,159 @@
+package org.sormas.e2etests.steps.web.application.immunizations;
+
+import static org.sormas.e2etests.pages.application.events.CreateNewEventPage.START_DATA_INPUT;
+import static org.sormas.e2etests.pages.application.immunizations.CreateNewImmunizationPage.*;
+
+import cucumber.api.java8.En;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.TextStyle;
+import java.util.Locale;
+import javax.inject.Inject;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+import org.sormas.e2etests.pojo.web.Immunization;
+import org.sormas.e2etests.services.ImmunizationService;
+
+public class CreateNewImmunizationSteps implements En {
+
+  public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("M/d/yyyy");
+  public static Immunization immunization;
+  private final WebDriverHelpers webDriverHelpers;
+
+  @Inject
+  public CreateNewImmunizationSteps(
+      WebDriverHelpers webDriverHelpers, ImmunizationService immunizationService) {
+    this.webDriverHelpers = webDriverHelpers;
+
+    When(
+        "^I create a new immunization with specific data$",
+        () -> {
+          immunization = immunizationService.buildGeneratedImmunization();
+          fillDateOfReport(immunization.getDateOfReport());
+          fillExternalId(immunization.getExternalId());
+          fillDisease(immunization.getDisease());
+          fillMeansOfImmunization(immunization.getMeansOfImmunization());
+          selectResponsibleRegion(immunization.getResponsibleRegion());
+          selectResponsibleDistrict(immunization.getResponsibleDistrict());
+          selectResponsibleCommunity(immunization.getResponsibleCommunity());
+          selectFacilityCategory(immunization.getFacilityCategory());
+          selectFacilityType(immunization.getFacilityType());
+          selectFacilityName(immunization.getFacility());
+          fillFacilityNameAndDescription(immunization.getFacilityDescription());
+          fillFirstName(immunization.getFirstName());
+          fillLastName(immunization.getLastName());
+          fillDateOfBirth(immunization.getDateOfBirth());
+          selectSex(immunization.getSex());
+          fillNationalHealthId(immunization.getNationalHealthId());
+          fillPassportNumber(immunization.getPassportNumber());
+          selectPresentConditionOfPerson(immunization.getPresentConditionOfPerson());
+          fillPrimaryPhoneNumber(immunization.getPrimaryPhoneNumber());
+          fillPrimaryEmailAddress(immunization.getPrimaryEmailAddress());
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
```

Comment:
after this save, create also the locator for Immunization Saved
 public static final By IMMUNIZATION_DATA_SAVED_POPUP = By.cssSelector(".v-Notification-caption");

And add:
webDriverHelpers.waitUntilElementIsVisibleAndClickable( IMMUNIZATION_DATA_SAVED_POPUP );
          webDriverHelpers.clickOnWebElementBySelector(CASE_SAVED_POPUP);
webDriverHelpers.clickOnWebElementBySelector(  IMMUNIZATION_DATA_SAVED_POPUP );

Reply:
Popup after the immunization creation doesn't show up so after adding this test fails.